test/build/50_ch-image.bats

```diff
@@ -31,6 +31,29 @@ setup () {
     [[ $output = *'verbose level: 1'* ]]
 }
 
+@test 'ch-image delete' {
+   run ch-image list
+   echo "$output"
+   [[ $status -eq 0 ]]
+   [[ $output != *"delete/test"* ]]
+  
+	ch-image build -t delete/test -f - . << 'EOF'
+FROM 00_tiny
+EOF
+	
+   run ch-image list
+	echo "$output"
+	[[ $status -eq 0 ]]
+	[[ $output = *"delete/test"* ]]
+	
+   ch-image delete delete%test
```

Comment:
This should take the image reference syntax, not the filesystem encoding:

```suggestion
   ch-image delete delete/test
```

Also, explain in a comment why you picked the test image name you did.